src/components/specialized/Achievements/AchievementsChain/index.js

```diff
@@ -166,15 +156,6 @@ class AchievementsChain extends Component {
               walletBalance={walletBalance}
               title={title}
             />
-            <Deposit
-              className={classes.actionsButtons}
-              link={object}
-              creatorID={creatorID}
-              creatorName={creatorName}
-              onDeposit={this.handleDeposit}
-              walletBalance={walletBalance}
-              title={title}
-            />
           </CardActions>
         ) : (
           <CardActions
```

Comment:
Same as previous comment, we cant just remove this parts if we want to still support Qtum